PACKAGE = "rm_forecast"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# select_bar = gen.enum([gen.const("Pass", int_t, 0, "Pass"),
#                        gen.const("Select", int_t, 1, "Select")], "type")
gen.add("max_match_distance", double_t, 0, "", 0.2, 0, 2)
gen.add("tracking_threshold", int_t, 0, "", 5, 0, 15)
gen.add("lost_threshold", int_t, 0, "", 5, 0, 15)

# gen.add("is_filter_readied", bool_t, 0, "", False)
# gen.add("min_lw_ratio", double_t, 0, "", 2.4, 0, 4)

exit(gen.generate(PACKAGE, "rm_forecast", "Forecast"))